packages/components/app/styles/components/code-block/index.scss

```diff
@@ -0,0 +1,252 @@
+      outline: 2px solid var(--token-color-focus-action-internal);
+      outline-offset: 1px;
+    }
+
+    &:hover {
+      color: var(--token-color-foreground-action-hover);
+    }
+
+    &:active {
+      color: var(--token-color-foreground-action-active);
+    }
+  }
+}
+
+.hds-code-block__body {
+  position: relative;
+}
+
+// Code
+.hds-code-block__code {
+  display: block;
+  margin: 0;
+  padding: $code-block-code-padding;
+  overflow: auto;
+  font-size: 0.8125rem;
+  font-family: var(--hds-typography-code-200);
+}
+
+// CopyButton
+.hds-code-block__copy-button {
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  // Overriding default colors
+  color: var(--hds-code-block-color-foreground-primary);
+  background-color: var(--hds-code-block-color-surface-faint);
+  border: 1px solid var(--hds-code-block-color-border-strong);
+
+  &:hover,
+  &.mock-hover {
+    background-color: var(--hds-code-block-color-surface-primary);
+    border-color: var(--hds-code-block-color-border-strong);
+  }
+
+  &:active,
+  &.mock-active {
+    background-color: var(--hds-code-block-color-surface-interactive-active);
+  }
+
+  &:focus,
+  &.mock-focus,
+  &:focus-visible {
+    background-color: var(--hds-code-block-color-surface-faint);
+  }
+
+  &.hds-copy-button--status-success {
+    .flight-icon {
+      color: var(--hds-code-block-color-foreground-success);
+    }
+  }
+
+  &.hds-copy-button--status-error {
+    .flight-icon {
+      color: var(--hds-code-block-color-foreground-critical);
+    }
+  }
+
+  .flight-icon { color: var(--hds-code-block-color-foreground-primary); }
+}
+
+// VARIANTS
+
+// isStandalone
+.hds-code-block--is-standalone {
+  border-radius: 6px;
+}
+
+// LineWrapping
+.hds-code-block--has-line-wrapping {
+  pre,
+  code { white-space: pre-wrap; }
+}
+
+// Linenumbers
+// Note: Prism.js is using the specific class name "line-numbers" to determine implementation of line numbers in the UI
+.hds-code-block {
+  // LineNumbers plugin styles ---------------
+  &.line-numbers {
+    counter-reset: linenumber;
+
+    .hds-code-block__code {
+      position: relative;
+      // reserve space for line numbers
+      padding-left: calc(#{$code-block-line-numbers-width} + #{$code-block-code-padding});
+    }
+
+    .line-numbers-rows {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: $code-block-line-numbers-width;
+      padding: $code-block-code-padding 0;
+      border-right: 1px solid var(--hds-code-block-color-border-primary);
+      user-select: none;
+      pointer-events: none;
+
+      > span {
+        display: block;
+        counter-increment: linenumber;
+
+        &::before {
+          display: block;
+          padding-right: $code-block-code-padding;
+          text-align: right;
+          content: counter(linenumber);
+        }
+      }
+    }
+  }
+
+  .line-numbers-rows .line-numbers-rows > span
+  // LineHighlight plugin styles (works with lineNumbers) ------------
+  pre[data-line] {
```

Comment:
This is [how it's been introduced](https://github.com/hashicorp/design-system/commit/45eed055727d6cc3f6db94f2948fdda139caa1e2#diff-6384d42404bdae907b5ef32f4cb2e302f98c4a608f086df699ad7a6705b0afeeR119) since the beginning.
I can move it one line up – probably that's where it belongs 🤷

Reply:
As it is now (`.line-numbers-rows .line-numbers-rows > span pre[data-line]` the selector doesn't match anything in the DOM.

This is the code I see in the downloaded `prism.css` so maybe that piece of CSS can be simply removed, since is not applied anyway, and the position relative and the padding is already applied elsewhere?
<img width="226" alt="screenshot_3207" src="https://github.com/hashicorp/design-system/assets/686239/1df9fc63-d63e-4c29-b43d-afb3674724b4">

also, looking at the code in `prism.css` there's no `.line-numbers-rows .line-numbers-rows > span` selector, so maybe something Kristin was playing with, and is simply a leftover.

Reply:
Removed the first line – I assume it was a leftover/remnant. It works fine just by targeting `span pre[data-line]` within the code section

Reply:
It was just left over and I had missed it when I last tried to clean up.